src/components/Composer/Editor/ToolbarPlugin.tsx

```diff
@@ -51,19 +49,13 @@ export default function ToolbarPlugin() {
       const elementDOM = activeEditor.getElementByKey(elementKey);
 
       if (elementDOM !== null) {
-        setSelectedElementKey(elementKey);
         if ($isListNode(element)) {
           const parentList = $getNearestNodeOfType<ListNode>(anchorNode, ListNode);
           const type = parentList ? parentList.getListType() : element.getListType();
           setBlockType(type);
         } else {
           const type = $isHeadingNode(element) ? element.getTag() : element.getType();
           setBlockType(type);
-          if ($isCodeNode(element)) {
```

Comment:
I left it thinking we might need this if we try to add code snippets

Reply:
Yup we can add incrementally 🙇🏼